packages/graph-explorer/src/components/Panel.tsx

```diff
@@ -152,7 +152,7 @@ function PanelHeaderActions({
   return (
     <div
       className={cn(
-        "flex grow flex-row items-center justify-end gap-0.5",
+        "flex grow flex-row items-center justify-end gap-0.5 overflow-x-auto",
```

Comment:
Since `PanelHeaderActions` is only a subset of the header, we don't need to modify its classes.